server/controllers/companyController.js

```diff
@@ -82,4 +89,46 @@ const updateCompany = async (req, res) => {
 }
 
 
-module.exports = { getCompanies, getCompany, createCompany, deleteCompany, updateCompany }
\ No newline at end of file
+//generating token function
+const createToken = (_id) => {
+
+    return jwt.sign({ _id }, process.env.SECRET, { expiresIn: '3d' })
```

Comment:
Most of the time you want to have expiresIn as an env, config or constants file, since you don't want to take a deep dive into the code if you want to change it later on, just fyi